lms/djangoapps/instructor_task/api.py

```diff
@@ -95,12 +99,10 @@ def submit_rescore_problem_for_student(request, usage_key, student):  # pylint:
     the problem doesn't support rescoring.
 
     This method makes sure the InstructorTask entry is committed.
-    When called from any view that is wrapped by TransactionMiddleware,
-    and thus in a "commit-on-success" transaction, an autocommit buried within here
-    will cause any pending transaction to be committed by a successful
-    save here.  Any future database operations will take place in a
-    separate transaction.
-
+    When called from any function that is wrapped in a transaction, an
+    autocommit buried within here will cause any pending transaction to
+    be committed by a successful save here. Any future database operations
+    will take place in a separate transaction.
```

Comment:
I'll add it to the documentation as well, but we need this because we need to commit the `InstructorTask` object before handing of the task to the queue. And that means a mid-request commit.